spec/acceptance/all_orders_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+RSpec.describe "#all_orders", vcr: { record: :once } do
+  let(:client) do
+    BinanceClient.new(
+      host: CONFIG[:host],
+      api_key: CONFIG[:sub_account_api_key],
+      api_secret: CONFIG[:sub_account_api_secret]
+    )
+  end
+
+  it "retrieves all orders within a timeframe" do
+    response = client.all_orders("ETHUSDT", "1651298308000")
+
+    expect(response).to be_success
```

Comment:
Because the README says "look at acceptance spec for usage", it's useful to see how its used here:
```suggestion
    expect(response).to be_success
    
    orders = response.orders
    expect(orders).to be_present
    order = orders.first
    
    aggregate_failures do
      expect(order.order_id).to be_a(Integer)
      expect(order.executed_quantity).to be_present
    end
```